src/main/java/edu/hm/hafner/analysis/parser/ClangAnalyzerPlistParser.java

```diff
@@ -0,0 +1,121 @@
+import java.util.List;
+
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.util.XmlElementUtil;
+
+
+public class ClangAnalyzerPlistParser extends IssueParser {
+    private static final long serialVersionUID = 1L;
+    static final String ID = "ClangAnalyzer Plist Parser";
+
+    @Override
+    public boolean accepts(final ReaderFactory readerFactory) {
+        return isXmlFile(readerFactory);
+    }
+
+    @Override
+    public Report parse(final ReaderFactory readerFactory) throws ParsingException {
+        try {
+            Document doc = readerFactory.readDocument();
+
+            XPathFactory xPathFactory = XPathFactory.newInstance();
+            XPath xPath = xPathFactory.newXPath();
+
+            XPathExpression filesPath = xPath.compile("/plist/dict/key[text()='files']/following-sibling::array/string");
+            XPathExpression diagnosticsPath = xPath.compile("/plist/dict/key[text()='diagnostics']/following-sibling::array/dict");
+
+            XPathExpression diagDescriptionPath = compileDiagStrPath(xPath, "description");
+            XPathExpression diagCategoryPath = compileDiagStrPath(xPath, "category");
+            XPathExpression diagTypePath = compileDiagStrPath(xPath, "type");
+            XPathExpression diagLocationLinePath = compileDiagLocationPath(xPath, "line");
+            XPathExpression diagLocationColPath = compileDiagLocationPath(xPath, "col");
+            XPathExpression diagLocationFilePath = compileDiagLocationPath(xPath, "file");
+
+
+            IssueBuilder issueBuilder = new IssueBuilder();
+            Report report = new Report();
+
+            List<String> filesList = new ArrayList<>();
+
+            // Fill files list
+
+            NodeList files = (NodeList)filesPath.evaluate(doc, XPathConstants.NODESET);
+            for (Element filePathStr : XmlElementUtil.nodeListToList(files)) {
+                filesList.add(filePathStr.getTextContent());
+            }
+
+            // Parse errors
+
+            NodeList diagnostics = (NodeList)diagnosticsPath.evaluate(doc, XPathConstants.NODESET);
+            for (Element diag : XmlElementUtil.nodeListToList(diagnostics)) {
+                issueBuilder
+                    .setFileName(filesList.get(extractIntField(diag, diagLocationFilePath)))
+                    .guessSeverity("Warning") //.guessSeverity(extractSeverity(problem))
+                    .setMessage(extractField(diag, diagDescriptionPath))
+                    .setLineStart(extractIntField(diag, diagLocationLinePath))
+					.setColumnStart(extractIntField(diag, diagLocationColPath))
+                    .setCategory(extractField(diag, diagCategoryPath))
+                    .setType(extractField(diag, diagTypePath))
+                    //.setAdditionalProperties()
+                    ;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [';' is preceded with whitespace.](https://app.codacy.com/manual/uhafner/analysis-model/pullRequest?prid=4824671)